### DISCUSSION TITLE
connection json error

### DISCUSSION THREAD

**Discussion by JeongHunP** [coinfoundry/miningcore#1359] [Q&A]
Hello, I have a problem connecting my local bfgminer with my miningcore pool.

I successfully connected my bitcoind with miningcore but when I opened miningcore pool and try to connect to this pool with bfgminer, it shows connection json error: [btc1] [0HMK35U5OFBNN] Connection json error: Unexpected character encountered while parsing value: . Path '', line 0, position 0.

![image](https://user-images.githubusercontent.com/38456845/185776159-e7642e28-96de-4eac-b1c9-055b4dc6dc51.png)

I tried to set address "127.0.0.1", 0.0.0.0" , "my ip address(which runs miningcore pool)" and use stratum+tcp://"those address":3052 at bfgminer.

Is there any solution?? Thanks.

**Comment by seperatis** [comment]
@JeongHunP 

Can you set `logLevel` to `debug` in your config? and run it again so we can get more detailed info?

It'd be helpful to see the output from that right after it accepts the connection, directly after the part you highlighted in Red

**Reply by JeongHunP** [reply]
@seperatis 
Oh, I can see it working after changing loglevel to debug! 
![image](https://user-images.githubusercontent.com/38456845/185802990-44df33eb-ffec-4f45-bf53-2c7794ea4a6a.png)
Authorize, subscribe .. etc , stratum processes are shown in the log, so does it mean mining pool correctly works?

**Comment by seperatis** [comment]
@JeongHunP 

It's not working correctly unless you are submitting shares. Is your miner submitting shares?
